configuration/configuration_blackbox_test.go

```diff
@@ -0,0 +1,108 @@
+package configuration
```

Comment:
Yes, it's a black box test but it shares the TestMain() function with the white box tests.